tests/bluetooth/bsim_bt/bsim_test_eatt/prj_multiple_conn.conf

```diff
@@ -9,6 +9,8 @@ CONFIG_BT_L2CAP_ECRED=y
 CONFIG_BT_EATT_MAX=5
 CONFIG_BT_MAX_CONN=2
 CONFIG_BT_EATT_AUTO_CONNECT=n
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+CONFIG_BT_CTLR_LLCP_LOCAL_PROC_CTX_BUF_NUM=4
```

Comment:
Make `CONFIG_BT_CTLR_LLCP_LOCAL_PROC_CTX_BUF_NUM=4`  as default in Kconfig. Setting `CONFIG_BT_CTLR_ADVANCED_FEATURES=y` in projects will display build warning.

Reply:
Done